### ISSUE TITLE
Add the ability to select which calendars should send notifications

### ISSUE THREAD

**Issue by GLLM** [nickv-nextcloud/event_update_notification#5]
Hello !

I like this other app of yours, and it works fine :) Many thanks !!

I'd love to be able to select the Agenda on which I'd like to be notified.
Indeed, my wife & I have several calendars ... all shared : one per person, and common and one for children.

Today, I get notifications for any action (Add/Update/Delete) in any of those agendas.

What I'd love would be to be able to activate the notification only for : shared & children agendas (notgetting notifs for her own agenda)

Could this be possible ?
I guess this would need to be user definable ...

Thanks
GLLM

**Comment by muppeth** [comment]
I work for small non-profut org where we just switched to nextcloud (yay). There is about 20 people and all of them share calendar with on another (in case they want to see availability of others etc). Although the calendars are disabled (when not needed) the notifications are pouring in rendering the calendar usage very hard to follow (you become immune to notifications as they are just too many and you miss your calendars notifs).

**Comment by mnobbes** [comment]
Dear developers team,
as the others in this thread I am using your app and I like it a lot. Thank you!
The mentioned feature would be a huge enhancement to your app. Are you still considering implementing it?
Bests
mnobbes

**Comment by nickvergessen** [comment]
Sadly there is no team, but it's me alone.
The problem is that this "little tiny config" explodes the efforts for this app. Currently it just listens to a single event and handles the notification. With this addition it needs a UI to select from a list of calendars, needs an APIP to store a setting, etc.

**Comment by mnobbes** [comment]
OK. Thank you for explaining and clarifying. 
All the best for your projects! :-)

**Comment by ghost** [comment]
I want this too.

A possible alternative to a separate UI would be a new item on the existing context menu for each calendar (where "Edit Name", "Edit Color", etc. live) to activate/deactivate notifications. Not being an NC developer, I don't know if that's easier or harder than a separate UI.

Thanks for the work.

**Comment by mate1213** [comment]
Hello,
Is there a plan to implement this feature request? I would be really happy, if this could be implemented in the near feature.

Thanks thanks for the dev team!

**Comment by nickvergessen** [comment]
I don't plan this at the moment as it introduces a settings and frontend section (not my expertise) which the app currently doesn't have.

Also I only use the app on my home instance and there all calendars are of interest.

**Comment by mate1213** [comment]
Thank you!
And your awesome work on this add-on!
I will try to solve this another way. 
(Maybe if the The calendars what I don't want to get notification will be in another profile, and I will disable the shared calendars notifications. This is also an add-on and not a base feature of nextcloud.)

**Comment by rostmodern** [comment]
> Hello !
> 
> I like this other app of yours, and it works fine :) Many thanks !!
> 
> I'd love to be able to select the Agenda on which I'd like to be notified. Indeed, my wife & I have several calendars ... all shared : one per person, and common and one for children.
> 
> Today, I get notifications for any action (Add/Update/Delete) in any of those agendas.
> 
> What I'd love would be to be able to activate the notification only for : shared & children agendas (notgetting notifs for her own agenda)
> 
> Could this be possible ? I guess this would need to be user definable ...
> 
> Thanks GLLM

Older isssue, still we face a similar situation and a finer grained possibility for notifications (i.e. per calendar) would be super helpful!
